addon/helpers/native.js

```diff
@@ -0,0 +1,15 @@
+/**
```

Comment:
Unless this file is used by the app (outside of tests), it should live in addon-test-support

Reply:
Interestingly, `drag` and `reorder` are also not used by the app.
`waiter` also seems to be used only by the test, but is wrapped inside a `DEBUG` flag in the app.
```
document.dispatchEvent(new Event('ember-sortable-drop-stop')
```
Sounds like they all can be moved into `addon-test-support`?

Reply:
I would say yes except that would require a major version bump and complicates your PR. Prob best left for another time